### PR TITLE
feature gherking positive test board

### PR DESCRIPTION
**Pull request by oscarlopez8**

### CODE REVIEW

File: trello/features/functional/BoardPositive.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created by Admin at 2/3/2020
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardPositive.feature]: add poyur name please.

File: trello/features/functional/BoardPositive.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Feature: Create a new Board with and without label
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardPositive.feature]: improve this comment to description

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+json
+re
+Pytest
+
+Go to https://developers.trello.com/reference/ to read Trello API reference
+Go to https://behave.readthedocs.io/en/stable/ if you can check the behave reference.
+
+......
+
+
```
Comment (JuanCarlosMartinez4) [inline README.md]: improve this Readme file, using tags to do a better job.

File: trello/features/functional/BoardPositive.feature [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      | key  | value          |
+      | name | newBoard       |
+      | desc | newDescription |
+    When Sends request
+    Then Should return status code 200
+    And Saves response as "BoardObject"
+    And Validates response body
+    And Validates schema with "board_schema.json"
+    #Post Condition
+    And Sets a "DELETE" request to "/boards/boardObject.id"
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardPositive.feature]: BoardObject improve this line of code

File: README.md (hunk 3 above)
Comment (JuanCarlosMartinez4) [inline README.md]: delete this readme file

File: trello/features/functional/BoardPositive.feature [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Created by Oscar Lopez at 2/3/2020
+Feature: Create a new Board with name and description
+  As a regular user, Manages the board, and user Creates a board with description.
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardPositive.feature]: add the object

File: trello/features/functional/BoardPositive.feature [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Created by Oscar Lopez at 2/3/2020
+Feature: Create a new Board with name and description
+  As a regular user, it wants to manage a board, and user Creates a board with description.
```
Comment (JuanCarlosMartinez4) [inline trello/features/functional/BoardPositive.feature]: improve this description to "and creates ..."

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Review by Amet015 — APPROVED** [review]

**Pull request merged**